RBAC for dynamic configuration documentation (#17861)
netdata/netdata: docs/netdata-agent/configuration/dynamic-configuration.md

**Cloud Connection and Security**: Nodes using Dynamic Configuration Manager require a connection to Netdata Cloud. This ensures proper permission handling and data security.

> **Info**
>

> > For details on what **permissions** apply based on a user's role, please refer to the [Role Based Access documentation](/docs/netdata-cloud/authentication-and-authorization/role-based-access-model.md#dynamic-configuration-manager) 

Reviewer: 1. I noticed that for some reason you often try to put some information in blockquotes. Why? You should not do this with important information.
2. I don't think we need to use "please" in our docs, should be more formal.

Author: 1. It was some time ago that we said notes and info looked better in blockquotes because they stand out in learn more than bold text, especially in that section, there is a lot of bold text.
2. Agree, I will remove it

> > For details on what **permissions** apply based on a user's role, please refer to the [Role Based Access documentation](/docs/netdata-cloud/authentication-and-authorization/role-based-access-model.md#dynamic-configuration-manager) 

Reviewer: ```suggestion
> To understand what actions users can perform based on their role, refer to the [Role Based Access documentation](https://github.com/docs/netdata-cloud/authentication-and-authorization/role-based-access-model.md#dynamic-configuration-manager). 
```

Author: @ilyam8 https://github.com/docs/netdata-cloud/authentication-and-authorization/role-based-access-model.md#dynamic-configuration-manager 😆

> > To understand what actions users can perform based on their role, refer to the [Role Based Access documentation](https://github.com/docs/netdata-cloud/authentication-and-authorization/role-based-access-model.md#dynamic-configuration-manager). 

Reviewer: ```suggestion
> To understand what actions users can perform based on their role, refer to the [Role Based Access documentation](/docs/netdata-cloud/authentication-and-authorization/role-based-access-model.md#dynamic-configuration-manager). 
```
